Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+# Check out https://hub.docker.com/_/node to select a new base image
+FROM node:16-alpine
+# Install dependencies
+# RUN npm install -g nodemon
+# RUN npm i -g @loopback/cli
+# Set to a non-root built-in user `node`
```

Comment:
On the Dockerfile updates, please consider commenting out unused code or even better, remove them. The `npm install -g nodemon`, `npm i -g @loopback/cli`, `ENV PATH /app/node_modules/.bin:$PATH`, `COPY --chown=node server/package*.json ./`, `RUN npm install`, `COPY --chown=node server/ .` are all commented out.